content/guides/exporters/supported-exporters/Java/stackdriver-stats.md

```diff
@@ -0,0 +1,167 @@
+import io.opencensus.stats.View.Name;
+import io.opencensus.stats.ViewManager;
+import io.opencensus.tags.TagKey;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+
+/** StackdriverExample is an example of exporting a custom metric from OpenCensus to Stackdriver. */
+public final class StackdriverExample {
+
+  // The task latency in milliseconds.
+  private static final MeasureDouble LATENCY_MS =
+      MeasureDouble.create("task_latency", "The task latency in milliseconds", "ms");
+
+  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
+
+  /** Main launcher for the Stackdriver example. */
+  public static void main(String... args) throws IOException, InterruptedException {
+    
+    // Defining the distribution aggregations
+    Aggregation latencyDistribution =
+        Distribution.create(
+            BucketBoundaries.create(
+                Arrays.asList(
+                    // Latency in buckets: [>=0ms, >=100ms, >=200ms, >=400ms, >=1s, >=2s, >=4s]
+                    0.0, 100.0, 200.0, 400.0, 1000.0, 2000.0, 4000.0)));
+     View view =
+        View.create(
+            Name.create("task_latency_distribution"),
+            "The distribution of the task latencies",
+            LATENCY_MS,
+            latencyDistribution,
+            Collections.<TagKey>emptyList());
+
+    // Create the view manager
+    ViewManager vmgr = Stats.getViewManager();
+
+    // Then finally register the views
+    vmgr.registerView(view);
+
+    // Enable OpenCensus exporters to export metrics to Stackdriver Monitoring.
+    // Exporters use Application Default Credentials to authenticate.
+    // See https://developers.google.com/identity/protocols/application-default-credentials
+    // for more details.
+    // The minimum reporting period for Stackdriver is 1 minute.
+    StackdriverStatsExporter.createAndRegister();
+
+    // Record 100 fake latency values between 0 and 5 seconds.
+    for (int i = 0; i < 100; i++) {
+      double ms = Math.random() * 5 * 1000;
+      System.out.println(String.format("Latency %d: %f", i, ms));
+      statsRecorder.newMeasureMap().put(LATENCY_MS, ms).record();
+      Thread.sleep(1000);
+    }
+    
+    System.out.println("Done recording metrics");
+  }
+}
+
+{{</highlight>}}
+
+{{<highlight xml>}}
+Insert the following snippet in your `pom.xml`:
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <opencensus.version>0.16.0</opencensus.version> <!-- The OpenCensus version to use -->
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-api</artifactId>
+            <version>${opencensus.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.opencensus</groupId>
+            <artifactId>opencensus-impl</artifactId>
+            <version>${opencensus.version}</version>
```

Comment:
nit: `<scope>runtime</scope>`

Reply:
Done. We need to change this on other places also.